easybuild/easyconfigs/p/pangolin/pangolin-4.0.6-foss-2021b-pangolin-data-1.3.eb

```diff
@@ -0,0 +1,75 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# Based on a EC provided by Paul Jähne
+# uploaded by J. Sassmannshausen
+# using the pangoLEARN extension to distinguish between different versions
```

Comment:
This comment now seems a bit out of date?

Reply:
@verdurin I updated it to `pangolin-data`

Reply:
Thanks.